README.md

```diff
@@ -63,15 +62,6 @@ Kapta requires a configuration file to be created in the src directory. The file
   }
 }
 ```
-
-## Usage
```

Comment:
why remove this section?

Reply:
It is redundant with the "What?" section, isn't?